src/main/java/uk/gov/hmcts/ccd/v2/V2.java

```diff
@@ -38,6 +39,7 @@ private MediaType() {}
         private Error() {}
 
         public static final String CASE_NOT_FOUND = "Case not found";
+        public static final String PRINTABLE_DOCUMENTS_ENDPOINT_DOWN = "Print documents endpoint is down";
```

Comment:
Do we really need to communicate this fact in a http-500 response? It sounds like a unnecessary info exposure, if not a super-harmful one.